htmlToElement.js

```diff
@@ -84,7 +84,9 @@ export default function htmlToElement(rawHtml, opts, done) {
 
         let listItemPrefix = null;
         if (node.name == 'li') {
-          if (parent.name == 'ol') {
+          if (!parent) {
+            listItemPrefix = BULLET;
```

Comment:
See changes to this code https://github.com/jsdf/react-native-htmlview/blob/master/htmlToElement.js#L109-L116 ... PR will need minor updating to accommodate this. Thanks!